src/index.html

```diff
@@ -22,6 +22,401 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="navigation">
+      <div class="navigation_logo">
+        <a href="#">
+          <img
+            src="images\logo.png"
+            alt="logo"
+          />
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__body">
+          <div class="card__logo">
+            <img
+              src="/src/images/imac.jpeg"
+              alt="product-img"
+            />
+          </div>
+          <h1 class="card__text">
```

Comment:
```suggestion
          <h2 class="card__text">
```
The `h1` title should be used once per page